vba/examples/example8_lineNumbers.bas

```diff
@@ -9,4 +9,5 @@ Dim Number, MyString
  MyString = "Number equals 2" 
 3: 
  Debug.Print MyString
+4: Debug.Print "TEST"
```

Comment:
This change doesn't test the changed grammar rule "lineLabel". There's not a single test that has "lineLabel".

```
01/26-05:00:52 ~/issues/g4-3880/vba/Generated-CSharp
$ trparse ../examples/*.bas | trxgrep ' //lineLabel' | trtext -c
CSharp 0 ../examples/example09_typeHints.bas success 0.3759407
CSharp 0 ../examples/example1.bas success 0.0414117
CSharp 0 ../examples/example2operators.bas success 0.1742185
CSharp 0 ../examples/example3erase.bas success 0.0385953
CSharp 0 ../examples/example4fixedstring.bas success 0.006915
CSharp 0 ../examples/example5property.bas success 0.0179688
CSharp 0 ../examples/example6dateliteral.bas success 0.0156073
CSharp 0 ../examples/example7linecontinuation.bas success 0.3410573
CSharp 0 ../examples/example8_lineNumbers.bas success 0.3330462
../examples/example09_typeHints.bas:0
../examples/example1.bas:0
../examples/example2operators.bas:0
../examples/example3erase.bas:0
../examples/example4fixedstring.bas:0
../examples/example5property.bas:0
../examples/example6dateliteral.bas:0
../examples/example7linecontinuation.bas:0
../examples/example8_lineNumbers.bas:0
01/26-05:03:05 ~/issues/g4-3880/vba/Generated-CSharp
$
```

Here's a "coverage heat map" looks like for the tests (rename to cover.html and view in an internet browser). 
[cover.html.txt](https://github.com/antlr/grammars-v4/files/14063316/cover.html.txt)

Reply:
I was wondering  if there was a way to see coverage. Unfortunately, removing the ‘.txt’ just gives a 404 error.

Reply:
Download cover.html.txt (I just tried the link, it works). Save it on disk. Rename cover.html.txt to cover.html in a command-line shell. Open a browser and click to open cover.html.

Reply:
@kaby76 Do you know if there is a way to see the coverage file from a working branch on GitHub? I’m adding examples and want to see which tokens I’m missing along the way.
I’m used to sending coverage reports to coveralls.io.
Thanks!

Reply:
Coverage is not run in the Github Actions build because it is time consuming. https://github.com/antlr/grammars-v4/pull/3624

You'll have to run it locally on your machine.